<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Zachary Winn - Publications</title>
    <link rel="stylesheet" href="static/style.css">
    <style>
        .year-banner {
            background-color: #000000; /* black background */
            color: white; /* White text */
            font-size: 1.5em; /* Adjust text size */
            padding: 10px; /* Add padding */
            text-align: center; /* Center text */
            margin-top: 30px; /* Space before year banner */
            margin-bottom: 10px; /* Space after year banner */
        }
    </style>
</head>
<body>
    <header>
        <h1>Zachary James Winn, PhD</h1>
        <p>Plant Breeder, Quantitative Geneticist, Bioinformatician, and Data Analyst</p>
    </header>

    <section>
        <h2>Publications</h2>
        <p>Click on publication titles for a download of the relevant PDF.</p>
        
        <div class="year-banner">2025</div>
        <div>
            <h3><a href="static/2025 - Crop Science - Winn - Genetic mapping of resistance to Fusarium head blight in soft red winter wheat line NC13‚Äê20076.pdf">Genetic mapping of resistance to Fusarium head blight in soft red winter wheat line NC13‚Äê20076</a></h3>
            <h4><strong>Abstract:</strong></h4>
            <p style="text-align: justify;"> Fusarium head blight (FHB) infection causes yield loss, quality degradation, and the production of damaging mycotoxins in common wheat (Triticum aestivum L). Marker analysis suggests that NC13-20076 does not possess previously identified FHB resistance quantitative trait loci (QTL) screened for in eastern winter wheat germplasm. A doubled haploid population of 168 lines from the cross of GA06493-13LE6 and NC13-20076 was phenotyped in inoculated nurseries in six environments. Heading date, plant height, and visual ratings of Fusarium damage on heads were recorded in the field; percent Fusarium damaged kernels (FDK) and deoxynivalenol (DON) accumulation were recorded post-harvest. Interval and multiple QTL mapping were performed on each environment-by-trait combination. Plant height and heading date QTL were identified on chromosomes 4A, 5A, 6A, and 7B, and peak markers were used as covariates in mapping of disease response traits. Disease response QTL were identified on chromosomes 1A, 2A, 2B, 3A, 3B, 4A, 5A, 7A, and 7D. The largest percent variance (PV) QTL identified for FHB visual ratings (10.8%) and DON accumulation (10.1%) were found on chromosome 5A (QFvr.nc-5A, QDon.nc-5A). The largest PV (10.3%) QTL identified for FDK were found on 1A (QFdk.nc-1A). Disease response QTL for multi-environment scans of visual ratings, FDK, and DON accumulation accounted for 4.0%‚Äì10.8%, 4.1%‚Äì10.3%, and 4.9%‚Äì10.1% of the total variance, respectively. The present results indicate that NC13-20076 contains several FHB response QTL, which overlap with previously identified QTL and demonstrate the importance of NC13-20076 as a readily accessible source of FHB resistance. </p>
        </div>

        <div class="year-banner">2024</div>
        <div>
            <h3><a href="static/2024 - Crop Science - Wondifraw - Advancing water absorption capacity in hard winter wheat using a multivariate genomic.pdf"> Advancing water absorption capacity in hard winter wheat using a multivariate genomic prediction approach</a></h3>
            <h4><strong>Abstract:</strong></h4>
            <p style="text-align: justify;"> The water absorption capacity (WAC) of hard wheat (Triticum aestivum L.) flour affects end-use quality characteristics, including loaf volume, bread yield, and shelf life. However, improving WAC through phenotypic selection is challenging. Phenotyping for WAC is time consuming and, as such, is often limited to evaluation in the latter stages of the breeding process, resulting in the retention of suboptimal lines longer than desired. This study investigates the potential of univariate and multivariate genomic predictions as an alternative to phenotypic selection for improving WAC. A total of 497 hard winter wheat genotypes were evaluated in multi-environment advanced yield and elite trials over 8 years (2014‚Äì2021). Phenotyping for WAC was done via the solvent retention capacity (SRC) using water as a solvent (SRC-W). Traits that exhibited a significant correlation (r ‚â• 0.3) with SRC-W and were evaluated earlier than SRC-W were included in the multivariate genomic prediction models. Kernel hardness and diameter were obtained using the single kernel characterization system (SKCS), and break flour yield and total flour yield (T-Flour) were included. Cross-validation showed the mean univariate genomic prediction accuracy of SRC to be r = 0.69 ¬± 0.005, while bivariate and multivariate models showed an improved prediction accuracy of r = 0.82 ¬± 0.003. Forward validation showed a prediction accuracy up to r = 0.81 for a multivariate model that included SRC-W + All traits (SRC-W, Diameter, SKCS hardness and diameter, F-Flour, and T-Flour). These results suggest that incorporating correlated traits into genomic prediction models can improve early-generation prediction accuracy. </p>
        </div>
        <div>
            <h3><a href="static/2024 - The Plant Genome - Wondifraw - Elucidation of the genetic architecture of water absorption capacity in hard winter.pdf">Elucidation of the genetic architecture of water absorption capacity in hard winter wheat through genome wide association study</a></h3>
            <h4><strong>Abstract:</strong></h4>
            <p style="text-align: justify;"> The water absorption capacity (WAC) of hard wheat (Triticum aestivum L.) flour affects end-use quality characteristics, including loaf volume, bread yield, and shelf life. However, improving WAC through phenotypic selection is challenging. Phenotyping for WAC is time consuming and, as such, is often limited to evaluation in the latter stages of the breeding process, resulting in the retention of suboptimal lines longer than desired. This study investigates the potential of univariate and multivariate genomic predictions as an alternative to phenotypic selection for improving WAC. A total of 497 hard winter wheat genotypes were evaluated in multi-environment advanced yield and elite trials over 8 years (2014‚Äì2021). Phenotyping for WAC was done via the solvent retention capacity (SRC) using water as a solvent (SRC-W). Traits that exhibited a significant correlation (r ‚â• 0.3) with SRC-W and were evaluated earlier than SRC-W were included in the multivariate genomic prediction models. Kernel hardness and diameter were obtained using the single kernel characterization system (SKCS), and break flour yield and total flour yield (T-Flour) were included. Cross-validation showed the mean univariate genomic prediction accuracy of SRC to be r = 0.69 ¬± 0.005, while bivariate and multivariate models showed an improved prediction accuracy of r = 0.82 ¬± 0.003. Forward validation showed a prediction accuracy up to r = 0.81 for a multivariate model that included SRC-W + All traits (SRC-W, Diameter, SKCS hardness and diameter, F-Flour, and T-Flour). These results suggest that incorporating correlated traits into genomic prediction models can improve early-generation prediction accuracy. </p>
        </div>

        <div class="year-banner">2023</div>
        <div>
            <h3><a href="static/2023 - Agronomy Journal - Winn - Yield protection afforded by imidacloprid during Hessian fly infestation in six genotypes.pdf">Yield protection afforded by imidacloprid during Hessian fly infestation in six genotypes</a></h3>
            <h4><strong>Abstract:</strong></h4>
            <p style="text-align: justify;"> The Hessian fly (Mayetiola destructor Say) is a gall midge that infests and feeds upon wheat (Triticum aestivum L.). Recently, a new form of tolerance (QHft.nc-7D) was identified in the breeding line LA03136E71 (PI 700336). Partial resistance allows immature Hessian fly to thrive in small numbers and does not function like antibiosis. Little is known about the potential yield drag of using partial resistance. In this study, we evaluated six genotypes: one containing QHft.nc-7D (LA03136E71), one containing H13, and four potentially susceptible genotypes. All genotypes were evaluated with two different seed treatment regiments of imidacloprid: no treatment and a two times rate of imidacloprid. All tested genotypes were planted in six-to-eight replications of a full factorial design in four environments. Subsamples of yield trial plots were taken to measure percent infested tillers and a number of larvae/pupae per tiller. Plots were harvested and grain yield was recorded. There was a significant (p[>F] < 0.05) reduction of percent infested tillers and a number of larvae/pupae per tiller related to the imidacloprid treatment. Imidacloprid treatment significantly (p[>T] < 0.05) reduced the number of larvae/pupae per tiller for LA03136E71. There was no significant (p[>T] > 0.05) grain yield increase associated with treatment for LA03136E71. This indicates that a two times application of imidacloprid on LA03136E71 (QHft.nc-7D) did not improve yield yet reduced infestation. Therefore, QHft.nc-7D, while allowing Hessian fly to thrive on the plant, does not significantly compromise yield.</p>
        </div>
        <div>
            <h3><a href="static/2023 - Plant Breeding - Boyles - Approaching 25 years of progress towards Fusarium head blight resistance in southern soft.pdf">Approaching 25 years of progress towards Fusarium head blight resistance in southern soft red winter wheat</a></h3>
            <h4><strong>Abstract:</strong></h4>
            <p style="text-align: justify;"> Tremendous progress has been made in variety development and host plant resistance to mitigate the impact of Fusarium head blight (FHB) since the disease manifested in the southeastern United States in the early 2000s. Much of this improvement was made possible through the establishment of and recurring support from the US Wheat & Barley Scab Initiative (USWBSI). Since its inception in 1997, the USWBSI has enabled land-grant institutions to make advances in reducing the annual threat of devastating FHB epidemics. A coordinated field phenotyping effort for annual germplasm screening has become a staple tool for selection in public and private soft red winter wheat (SRWW) breeding programmes. Dedicated efforts of many SRWW breeders to identify and utilize resistance genes from both native and exotic sources provided a strong foundation for improvement. In recent years, implementation of genomics-enabled breeding has further accelerated genetic gains in FHB resistance. This article reflects on the improvement of FHB resistance in southern SRWW and contextualizes the monumental progress made by collaborative, persistent, and good old-fashioned cultivar development.</p>
        </div>
        <div>
            <h3><a href="static/2023 - The Plant Genome - Winn - HaploCatcher  An R package for prediction of haplotypes.pdf">HaploCatcher: An R package for prediction of haplotypes</a></h3>
            <h4><strong>Abstract:</strong></h4>
            <p style="text-align: justify;"> Wheat (Triticum aestivum L.) is crucial to global food security but is often threatened by diseases, pests, and environmental stresses. Wheat-stem sawfly (Cephus cinctus Norton) poses a major threat to food security in the United States, and solid-stem varieties, which carry the stem-solidness locus (Sst1), are the main source of genetic resistance against sawfly. Marker-assisted selection uses molecular markers to identify lines possessing beneficial haplotypes, like that of the Sst1 locus. In this study, an R package titled ‚ÄúHaploCatcher‚Äù was developed to predict specific haplotypes of interest in genome-wide genotyped lines. A training population of 1056 lines genotyped for the Sst1 locus, known to confer stem solidness, and genome-wide markers was curated to make predictions of the Sst1 haplotypes for 292 lines from the Colorado State University wheat breeding program. Predicted Sst1 haplotypes were compared to marker-derived haplotypes. Our results indicated that the training set was substantially predictive, with kappa scores of 0.83 for k-nearest neighbors and 0.88 for random forest models. Forward validation on newly developed breeding lines demonstrated that a random forest model, trained on the total available training data, had comparable accuracy between forward and cross-validation. Estimated group means of lines classified by haplotypes from PCR-derived markers and predictive modeling did not significantly differ. The HaploCatcher package is freely available and may be utilized by breeding programs, using their own training populations, to predict haplotypes for whole-genome sequenced early generation material.</p>
        </div>
        <div>
            <h3><a href="static/2023 - The Plant Genome - Winn - Utilization of a publicly available diversity panel in genomic prediction of Fusarium head.pdf">Utilization of a publicly available diversity panel in genomic prediction of Fusarium head blight resistance traits in wheat</a></h3>
            <h4><strong>Abstract:</strong></h4>
            <p style="text-align: justify;"> Fusarium head blight (FHB) is an economically and environmentally concerning disease of wheat (Triticum aestivum L). A two-pronged approach of marker-assisted selection coupled with genomic selection has been suggested when breeding for FHB resistance. A historical dataset comprised of entries in the Southern Uniform Winter Wheat Scab Nursery (SUWWSN) from 2011 to 2021 was partitioned and used in genomic prediction. Two traits were curated from 2011 to 2021 in the SUWWSN: percent Fusarium damaged kernels (FDK) and deoxynivalenol (DON) content. Heritability was estimated for each trait-by-environment combination. A consistent set of check lines was drawn from each year in the SUWWSN, and k-means clustering was performed across environments to assign environments into clusters. Two clusters were identified as FDK and three for DON. Cross-validation on SUWWSN data from 2011 to 2019 indicated no outperforming training population in comparison to the combined dataset. Forward validation for FDK on the SUWWSN 2020 and 2021 data indicated a predictive accuracy r ‚âà 0.58 and r ‚âà 0.53, respectively. Forward validation for DON indicated a predictive accuracy of r ‚âà 0.57 and r ‚âà 0.45, respectively. Forward validation using environments in cluster one for FDK indicated a predictive accuracy of r ‚âà 0.65 and r ‚âà 0.60, respectively. Forward validation using environments in cluster one for DON indicated a predictive accuracy of  r ‚âà 0.67 and r ‚âà 0.60, respectively. These results indicated that selecting environments based on check performance may produce higher forward prediction accuracies. This work may be used as a model for utilizing public resources for genomic prediction of FHB resistance traits across public wheat breeding programs..</p>
        </div>
        <div>
            <h3><a href="static/2023 - The Plant Phenome Journal - Hammers - Phenotyping and predicting wheat spike characteristics using image analysis.pdf">Phenotyping and predicting wheat spike characteristics using image analysis and machine learning</a></h3>
            <h4><strong>Abstract:</strong></h4>
            <p style="text-align: justify;"> Improvements in trait phenotyping are needed to increase the quantity and quality of data available for genetic improvement of crops. In this study, we used moderate throughput image analysis and machine learning as a pipeline for phenotyping a key wheat spike characteristic: spikelet number per spike. A population of 594 soft red winter wheat inbred lines was evaluated in the field for 2 years and images of wheat spikes were taken and used to train deep-learning algorithms to predict spikelet number. A total of 12,717 images were used to train, test, and validate a basic regression convolutional neural network (CNN), a visual geometry group application regression model, VGG16, the ResNet152V2 model, and the EfficientNetV2L model. The EfficientNetV2L model was the most accurate, having the lowest mean absolute error, second lowest root mean square error, and highest coefficient of determination (mean absolute error [MAE] = 0.60, root mean square error [RMSE] = 0.79, and R2 = 0.90). The ResNet152V2 model was slightly less accurate with a slightly better fit (MAE = 0.61,m RMSE = 0.78, and R2 = 0.87), followed by the basic CNN (MAE = 0.75, RMSE = 1.00, and R2 = 0.74) and finally by the VGG16 (MAE = 1.51, RMSE = 1.29, and R2 = 0.076). With an average error of just above one half of a spikelet, utilizing image analysis and machine learning counting methods could be used for multiple breeding applications, including direct selection of spikelet number, to provide data to identify quantitative trait loci, or for training whole genome selection models.</p>
        </div>
        <div>
            <h3><a href="static/2023 - The Plant Phenome Journal - Winn - Phenomic versus genomic prediction A comparison of prediction accuracies for.pdf">Phenomic versus genomic prediction‚ÄîA comparison of prediction accuracies for grain yield in hard winter wheat lines</a></h3>
            <h4><strong>Abstract:</strong></h4>
            <p style="text-align: justify;"> Common bread wheat (Triticum aestivum L.) is a key component of global diets, but the genetic improvement of wheat is not keeping pace with the growing demands of the world's population. To increase efficiency and reduce costs, breeding programs are rapidly adopting the use of unoccupied aerial vehicles to conduct high-throughput spectral analyses. This study examined the effectiveness of multispectral indices in predicting grain yield compared to genomic prediction. Multispectral data were collected on advanced generation yield nursery trials during the 2019‚Äì2021 growing seasons in the Colorado State University Wheat Breeding Program. Genome-wide genotyping was performed on these advanced generations and all plots were harvested to measure grain yield. Two methods were used to predict grain yield: genomic estimated breeding values (GEBVs) generated by a genomic best linear unbiased prediction (gBLUP) model and phenomic phenotypic estimates (PPEs) using only spectral indices via multiple linear regression (MLR), k-nearest neighbors (KNNs), and random forest (RF) models. In cross-validation, PPEs produced by MLR, KNN, and RF models had higher prediction accuracy (r:r 0.41 < r < 0.48) than GEBVs produced by gBLUP (r = 0.35). In leave-one-year-out forward validation using only multispectral data for 2020 and 2021, PPEs from MLR and KNN models had higher prediction accuracy of grain yield than GEBVs of those same lines. These findings suggest that a limited number of spectra may produce PPEs that are more accurate than or equivalently accurate as GEBVs derived from gBLUP, and this method should be evaluated in earlier development material where sequencing is not feasible.</p>
        </div>
        <div>
            <h3><a href="static/2023 -Crop Science - Winn - Multivariate genomic selection models improve prediction accuracy of agronomic traits in soft.pdf">Multivariate genomic selection models improve prediction accuracy of agronomic traits in soft red winter wheat</a></h3>
            <h4><strong>Abstract:</strong></h4>
            <p style="text-align: justify;"> Univariate genomic selection (UVGS) is an important tool for increasing genetic gain and multivariate GS (MVGS), where correlated traits are included in genomic selection, which can improve genomic prediction accuracy. The objectives for this study were to evaluate MVGS approaches to improve prediction accuracy for four agronomic traits using a training population of 351 soft red winter wheat (Triticum aestivum L.) genotypes, evaluated over six site-years in Arkansas from 2014 to 2017. Genotypes were phenotyped for grain yield, heading date, plant height, and test weight in both the training and test populations. In cross-validations, various combinations of traits in MVGS models significantly improved prediction accuracy for test weight in comparison to a UVGS model. Marginal increases in predictive accuracy were also observed for grain yield, plant height, and heading date. Multivariate models which were identified as superior to the univariate case in cross-validations were forward validated by predicting the advanced breeding nurseries of 2018 and 2020. In forward validation, consistent increases in accuracy were observed for test weight, plant height, and heading date using MVGS instead of UVGS. Overall, MVGS models improved prediction accuracies when correlated traits were included with the predicted response. The methods outlined in this study may be used to achieve higher prediction accuracies in unbalanced datasets over multiple environments.</p>
        </div>

        <div class="year-banner">2022</div>
        <div>
            <h3><a href="static/2022 - TAAG - Winn - Profiling of Fusarium head blight resistance QTL haplotypes through molecular markers, genotyping-by-sequencing, and machine learning.pdf">Profiling of Fusarium head blight resistance QTL haplotypes through molecular markers, genotyping-by-sequencing, and machine learning</a></h3>
            <h4><strong>Abstract:</strong></h4>
            <p style="text-align: justify;"> Marker-assisted selection is important for cultivar development. We propose a system where a training population genotyped for QTL and genome-wide markers may predict QTL haplotypes in early development germplasm. Breeders screen germplasm with molecular markers to identify and select individuals that have desirable haplotypes. The objective of this research was to investigate whether QTL haplotypes can be accurately predicted using SNPs derived by genotyping-by-sequencing (GBS). In the SunGrains program during 2020 (SG20) and 2021 (SG21), 1,536 and 2,352 lines submitted for GBS were genotyped with markers linked to the Fusarium head blight QTL: Qfhb.nc-1A, Qfhb.vt-1B, Fhb1, and Qfhb.nc-4A. In parallel, data were compiled from the 2011-2020 Southern Uniform Winter Wheat Scab Nursery (SUWWSN), which had been screened for the same QTL, sequenced via GBS, and phenotyped for: visual Fusarium severity rating (SEV), percent Fusarium damaged kernels (FDK), deoxynivalenol content (DON), plant height, and heading date. Three machine learning models were evaluated: random forest, k-nearest neighbors, and gradient boosting machine. Data were randomly partitioned into training-testing splits. The QTL haplotype and 100 most correlated GBS SNPs were used for training and tuning of each model. Trained machine learning models were used to predict QTL haplotypes in the testing partition of SG20, SG21, and the total SUWWSN. Mean disease ratings for the observed and predicted QTL haplotypes were compared in the SUWWSN. For all models trained using the SG20 and SG21, the observed Fhb1 haplotype estimated group means for SEV, FDK, DON, plant height, and heading date in the SUWWSN were not significantly different from any of the predicted Fhb1 calls. This indicated that machine learning may be utilized in breeding programs to accurately predict QTL haplotypes in earlier generations.</p>
        </div>

        <div class="year-banner">2021</div>
        <div>
            <h3><a href="static/2021 - TAAG - Winn - Mapping of a novel major effect Hessian fly field partial-resistance locus in southern soft red winter wheat line LA03136E71.pdf">Mapping of a novel major effect Hessian fly field partial-resistance locus in southern soft red winter wheat line LA03136E71</a></h3>
            <h4><strong>Abstract:</strong></h4>
            <p style="text-align: justify;"> Hessian fly resistance has centralized around resistance loci that are biotype specific. We show that field resistance is evident and controlled by a single locus on chromosome 7D. Hessian flies (Mayetiola destructor Say) infest and feed upon wheat (Triticum aestivum L) resulting in significant yield loss. Genetically resistant cultivars are the most effective method of Hessian fly management. Wheat breeders in the southern USA have observed cultivars exhibiting a "field resistance" to Hessian fly that is not detectable by greenhouse assay. The resistant breeding line "LA03136E71" and susceptible cultivar "Shirley" were crossed to develop a population of 200 random F4:5 lines using single seed descent. The population was evaluated in a total of five locations in North Carolina during the 2019, 2020, and 2021 seasons. A subsample of each plot was evaluated for the total number of tillers, number of infested tillers, and total number of larvae/pupae. From these data, the percent infested tillers, number of larvae/pupae per tiller, and the number of larvae/pupae per infested tiller were estimated. In all within and across environment combinations for all traits recorded, the genotype effect was significant (p < 0.05). Interval mapping identified a single large effect QTL distally on the short arm of chromosome 7D for all environment-trait combinations. This locus was identified on a chromosome where no other Hessian fly resistance/tolerance QTL has been previously identified. This novel Hessian fly partial-resistance QTL is termed QHft.nc-7D. Fine mapping must be conducted in this region to narrow down the causal agents responsible for this trait, and investigation into the mode of action is highly suggested.</p>
        </div>
        <div>
            <h3><a href="static/2021 - MDPI agronomy - Winn - Phenotyping Anther Extrusion of Wheat Using Image Analysis.pdf">Phenotyping Anther Extrusion of Wheat Using Image Analysis</a></h3>
            <h4><strong>Abstract:</strong></h4>
            <p style="text-align: justify;"> Phenotyping wheat (Triticum aestivum L.) is time-consuming and new methods are necessary to decrease labor. To develop a heterotic pool of male wheat lines for hybrid breeding, there must be an efficient way to measure both anther extrusion and the size of anthers. Five hundred and ninety-four soft red winter wheat lines in two replications of randomized complete block design were phenotyped for anther extrusion, a key trait for hybrid wheat production. A device was constructed to capture images using a mobile device. Four heads were sampled per line when anthesis was evident for half the heads in the plot. The extruded anthers were scraped onto a surface, their image was captured, and the area of the anthers was taken via ImageJ. The number of anthers extruded was estimated by counting the number of anthers per image and dividing by the number of heads sampled. The area per anther was taken by dividing the area of anthers per spike by the number of anthers per spike. A significant correlation (ùëÖ = 0.9, ùëù < 0.0001) was observed between the area of anthers per spike and the number of anthers per spike. This method is proposed as a substitute for field ratings of anther extrusion and to quantitatively measure the size of anthers.</p>
        </div>

        <div class="year-banner">2019</div>
        <div>
            <h3><a href="static/2019 - Euphytica - Hayat - Effects of allelic variation at Rht-B1 and Rht-D1 on grain yield and agronomic traits of southern US soft red winter wheat.pdf">Effects of allelic variation at Rht-B1 and Rht-D1 on grain yield and agronomic traits of southern US soft red winter wheat</a></h3>
            <h4><strong>Abstract:</strong></h4>
            <p style="text-align: justify;"> Semi dwarfism in hexaploid wheat (Triticum aestivum L.) is primarily governed by two loci, Rht-B1 and Rht-D1. Cultivars adapted to the soft red winter wheat growing region of southeastern USA are predominantly Rht-D1b genotypes but report no significant grain yield advantage over Rht-B1b semi dwarfing cultivars. The objective of this study was to determine the effect of allelic variation at Rht-B1 and Rht-D1 on plant height, grain yield and additional yield components in a doubled haploid population consisting of 35 semi dwarfs with Rht-B1b, 50 semi dwarfs with Rht-D1b, eight wild type lines, and two lines with dwarfing alleles at both loci. Rht loci significantly affected plant height, with double dwarfs shorter than both single gene semi dwarfs and wild types. Rht-D1b semi dwarfs were significantly shorter than their Rht-B1b counterparts. Rht loci also had a significant effect on grain yield, with Rht-D1b lines having higher mean grain yield (4.03 t ha<sup>‚àí1</sup>) compared to Rht-B1b (3.83 t ha<sup>‚àí1</sup>) and wild type (3.49 t ha<sup>‚àí1</sup>) lines. A significant interaction between Rht loci and site-year was detected only for thousand kernel weight, indicating that the advantage of Rht-D1b over the other haplotypes was consistent across environments. Overall, their higher grain yield was due in part to higher thousand kernel weight that contributed to higher kernel weight spike<sup>‚àí1</sup> and likely influenced by a shorter stature. The results of this study will aid breeders in choice of semi dwarfing alleles for adaptation to the soft wheat growing region of the southern USA.</p>
        </div>
    </section>

    <footer>
        <p>&copy; 2025 Zachary J. Winn</p>    
    </footer>
</body>
</html>